app/src/organisms/ChooseRobotSlideout/index.tsx

```diff
@@ -184,15 +184,18 @@ export function ChooseRobotSlideout(
     {}
   )
 
+  const reducerAvailable = healthyReachableRobots.filter(robot =>
```

Comment:
```suggestion
  const reducerAvailableRobots = healthyReachableRobots.filter(robot =>
```
Would be a bit easier to tell what this does if we specify "robots"